lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -139,7 +139,10 @@ namespace Js
     static char16 const funcName[] = _u("function anonymous");
     static char16 const genFuncName[] = _u("function* anonymous");
     static char16 const asyncFuncName[] = _u("async function anonymous");
-    static char16 const bracket[] = _u(" {\012");
+    static char16 const openFormals[] = _u("(");
+    static char16 const closeFormals[] = _u("\012)");
```

Comment:
you mean \n :)

It was already there, in this case I wouldn't perturb it.  Note @bterlson can't merge this in, so whoever signs off needs to merge it on his behalf, and take care of updating internal test baselines if any change. _/me backs away, runs_